src/components/overview/StyleSelector.jsx

```diff
@@ -2,41 +2,47 @@ import React, {useState} from 'react';
 import styles from './overview.module.css';
 import json from './styleSample.json';
 import {FaCheck} from 'react-icons/fa';
+import {StyleIndexContext} from './overview.jsx'
+
 
 const StyleSelector = (props) => {
-  var data = json;
-  var styleOptions = data.results //all results of styles form ID, sample data
-  const [selectedStyle, setStyle] = useState(styleOptions[0]);
 
+  console.log(props.styles)
```

Comment:
Sneaky console.logs trying to sneak into production code!

Reply:
Debugger's tend to sneak into mine.... I should write a program that works like pomander but it checks for debugger.

Reply:
Thanks for catching that. Will remove it.